app/controllers/user/sessions_controller.rb

```diff
@@ -1,8 +1,19 @@
 class User::SessionsController < Devise::SessionsController
 
   def login
-    flash[:notice] = "Signed in"
```

Comment:
not using it in that controller at the moment, using it in the users controller for when user sighs in successfully or not. Should this be done in the Sessions controller and not the users controller?